### ISSUE TITLE
Cannot import KonamiModule 1.7.0 with Angular 9

### ISSUE THREAD

**Issue by matcho** [lauthieb/ngx-konami#6]
Hi @lauthieb 

Thanks for merging PR #5 

However, now that I updated to 1.7.0 I cannot import KonamiModule anymore.

```TypeScript
import { KonamiModule } from "ngx-konami";
```
gives a
```
 error TS2307: Cannot find module 'ngx-konami'.
```

while 
```TypeScript
import { KonamiModule } from "ngx-konami/dist";
```
as mentioned in #2 gives a
```
ERROR in ./src/app/app.module.ngfactory.js
Module not found: Error: Can't resolve 'ngx-konami' in '/home/…/src/app'
```
When importing from `/dist`, the app is built but fails at startup with an empty page "Cannot GET /".

Any idea ? I tried completely removing the module and reinstalling it, with no success.

Thanks a lot

**Comment by BGBRWR** [comment]
The issue is that the NPM author published from the root of the project and not from the `dist` directory. The fix is for the NPM author to `CD` into the `dist` directory then publish.

**Comment by lauthieb** [comment]
Thanks folks, I will do this now.

**Comment by lauthieb** [comment]
I've just deprecated 1.7.0 version and published 1.7.1 version on NPM.
Thanks for reporting this.
